docs/template_plugin/tests/functional/op_reference/experimental_detectron_roi_feature_extractor.cpp

```diff
@@ -0,0 +1,236 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gtest/gtest.h>
+
+#include <ie_core.hpp>
+#include <ie_ngraph_utils.hpp>
+#include <ngraph/ngraph.hpp>
+#include <shared_test_classes/base/layer_test_utils.hpp>
+#include <tuple>
+
+#include "base_reference_test.hpp"
+
+using namespace ngraph;
+using namespace InferenceEngine;
+using namespace reference_tests;
+
+struct ExperimentalROIFunctional {
```

Comment:
Why do you need to have this structure?
I propose to remove it. Please have a look to proposal below.

Reply:
Fixed.